01-introduction.Rmd

```diff
@@ -8,10 +8,25 @@ By the end of the book you should be able to create new tools for geocomputation
 
 Over the last few decades free and open source software for geospatial data ('FOSS4G') has progressed at an astonishing rate (see [foss4g.org](http://foss4g.org/)).
 Thanks to FOSS4G and the wider open source movement geospatial analysis is no longer the preserve of those with expensive hardware and software: anyone can now download high performance spatial libraries on their computer.
+<!-- JM: I would delete this, since this is not really true.
 However, despite the growth of geospatial software that is *open source*, much of it is still not easy to script.
-Open source Geographic Information Systems (GIS) such as QGIS (see [qgis.org](http://qgis.org/en/site/)) have greatly reduced the 'barrier to entry' but their emphasis on the Graphical User Interface (GUI) can discourage reproducibility.
-This book focuses on the Command Line Interface (CLI), enabling reproducible, and 'computational' workflows, something we will expand on in Chapter 13.
+-->
+Open source Geographic Information Systems (GIS) such as QGIS (see [qgis.org](http://qgis.org/en/site/)) have greatly reduced the 'barrier to entry'.
+What is more, QGIS and other GIS software packages make command-line based processing easily accessible via application programming interfaces (API), something we will expand on in Chapter 13.
 <!--\@ref(gis) --><!-- REF NEEDS TO BE FIXED IN FUTURE-->
+Nevertheless, many users still prefer the Graphical User Interface (GUI) of GIS which in turn tends to discourage reproducibility (see Table \@ref(tab:gdsl)).
+This book focuses on the Command Line Interface (CLI) and how to use R as a GIS.
+This enables reproducible, and 'computational' workflows while making the most of R's statistical capabilities (see section \@ref(why-geocomputation-with-r)).
+
+<!--
```

Comment:
not sure if this table is needed. I have also modified the content a bit. AFAIK there is no GIS field, GIS has always been a software, a method, not a field used by several disciplines. Therefore, I now compare Desktop GUI GIS with R (to avoid apple-with-pear comparisons). I explicilty say Desktop GIS GUI since there are cli-based GIS and you can use virtually all Desktop GIS via the CLI. I think, we should point out the strengths of R ((geo-)statistics, data science, machine learning), and that it is therefore, a good idea to use R for geocomputation and geographic data science, respectively. In turn, we should not say, that GIS make cli-based programming difficult. It is not harder than learning R.

Reply:
Interesting point. GIS is certainly software but I think it is also a field of study with concepts, courses and more. At Leeds the MSc in GIS is one of the most popular from the School of Geography, for example (I hope you'll be able to visit at some point ; ): http://www.geog.leeds.ac.uk/study/masters/courses/msc-in-gis/ but I see your point that originally it was software. I'm happy with your change in any case but I would contend the table *is* useful, and will be more useful if we make it more about the 'GIS approach' vs the 'Gecomputation approach' - I think all can agree that they are approaches to solving problems regardless of views on whether or not they are fields, software or other.